server/config_load_test.go

```
@@ -0,0 +1,29 @@
+package chserver
+
+import (
+	"os"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	chshare "github.com/cloudradar-monitoring/rport/share"
+)
+
+func TestLoadingExampleConf(t *testing.T) {
+	var (
+		viperCfg *viper.Viper
+		cfg      = &Config{}
+	)
+	viperCfg = viper.New()
+	viperCfg.SetConfigType("toml")
+	viperCfg.SetConfigFile("../rportd.example.conf")
+	path, _ := os.Getwd()
```

You shouldn't ignore errors, but use `require.NoError` instead.